canonical/ubuntu.com
# /core/stories refresh

## Done

- Applied rebrand as per [doc](https://docs.google.com/document/d/1mXnYnpaCJznDLtYn9Q74aVeEI3iFxVOMzF0pdL7wdu0/edit) and [design](https://www.figma.com/file/QW6HKqDBbdNPdMfUaaEXV8/24.04-Core-Success-Stories?node-id=53-2047&mode=dev)

## QA

- View the site locally in your web browser at: https://ubuntu-com-13854.demos.haus/core/stories
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against doc and design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10478
